examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -60,31 +92,84 @@
     adhd.func[0], confounds=adhd.confounds[0])
 
 ###############################################################################
-# Extract and plot correlation matrix
+# Correlation matrices estimation
+# -------------------------------
+# All starts with estimation of the signals **covariance**. Here the number of
+# samples
+print(power_timeseries.shape[0])
+
+###############################################################################
+# is less than the number of ROIs for Power atlas
+# (and of the same order for Dosenbach atlas). In these situations,
+# graphical lasso **sparse inverse covariance** estimator captures
+# well the covariance **structure**.
+from sklearn.covariance import GraphLassoCV
+
+###############################################################################
+# This estimator can be encompassed into the connectivities estimator object
+# :class:`nilearn.connectome.ConnectivityMeasure`. We set the connectivity
+# *kind* to correlation, to normalize covariance coefficients. 
+
+from nilearn import connectome
+
+correlation_estimator = connectome.ConnectivityMeasure(
+    cov_estimator=GraphLassoCV(verbose=True), kind='correlation')
```

Comment:
If we're looking at a correlation matrix, using a GraphLasso rather than a LedoitWolf won't be a huge change. In the spirit of keeping this example simple, I suggest that we just stick to the default.